model/__tests__/lollipop_keys.test.ts

```diff
@@ -0,0 +1,162 @@
+import * as E from "fp-ts/lib/Either";
+import * as tk from "timekeeper";
+
+import { Container } from "@azure/cosmos";
+import { AssertionRef } from "@pagopa/io-functions-commons/dist/generated/definitions/lollipop/AssertionRef";
+import { AssertionTypeEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/lollipop/AssertionType";
+import { CosmosResource } from "@pagopa/io-functions-commons/dist/src/utils/cosmosdb_model";
+import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
+import { FiscalCode, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import {
+  AssertionFileName,
+  LolliPOPKeysModel,
+  PendingPopDocument,
+  LolliPopPubKeys,
+  RetrievedLolliPopPubKeys,
+  TTL_VALUE_AFTER_UPDATE,
+  TTL_VALUE_FOR_RESERVATION
+} from "../lollipop_keys";
+import { PubKeyStatusEnum } from "../../generated/definitions/internal/PubKeyStatus";
+
+const aFiscalCode = "RLDBSV36A78Y792X" as FiscalCode;
+const anAssertionRef = "sha256-p1NY7sl1d4lGvcTyYS535aZR_iJCleEIHFRE2lCHt-c" as AssertionRef;
+const anAssertionFileName = `${aFiscalCode}-${anAssertionRef}` as AssertionFileName;
+const aPubKey = "aValidPubKey" as NonEmptyString;
+
+const aCosmosResourceMetadata: Omit<CosmosResource, "id"> = {
+  _etag: "_etag",
+  _rid: "_rid",
+  _self: "_self",
+  _ts: 1
+};
+
+const aPendingPopDocument: PendingPopDocument = {
```

Comment:
I think it could be better to rename all xxxPopDocumentXXX to something like xxxLolliPopPubKeysXXX just to be more coherent with new naming.